edx_solutions_api_integration/courses/tests.py

```diff
@@ -1262,9 +1262,10 @@ def test_courses_users_list_courses_passed(self):
 
         response = self.do_get(test_uri.format(course_id=unicode(course.id)))
         self.assertEqual(response.status_code, 200)
-        self.assertEqual(len(response.data), 2)
-        self.assertEqual(response.data[0], users[0].id)
-        self.assertEqual(response.data[1], users[1].id)
+        self.assertEqual(response.data["count"], 2)
+
+        self.assertEqual(response.data["results"][0]["id"], users[0].id)
+        self.assertEqual(response.data["results"][1]["id"], users[1].id)
```

Comment:
I think we should assert presence of other user attributes here like email, username, first_name, last_name.